roles/webotsrobocup.role

```diff
@@ -0,0 +1,45 @@
+nuclear_role(
+  # FileWatcher, FileLogHandler and Signal Catcher Must Go First. KinematicsConfiguration usually goes after these and
+  # without it many roles do not run
+  extension::FileWatcher
+  support::SignalCatcher
+  support::logging::FileLogHandler
+  # This must come first as it emits config which many roles depend on (e.g. SensorFilter, WalkEngine)
+  motion::KinematicsConfiguration
+  # Support
+  support::configuration::GlobalConfig
+  network::NUClearNet
```

Comment:
Not sure we need NUClearNet? Unless we plan on trying to organise inter-robot communication? @TrentHouliston

Reply:
Commenting out for now.